index.html

```diff
@@ -211,5 +211,36 @@ <h1>About me</h3>
             </div>
         </div>
     </section>
+    <section class="contact-form">
+        <div class="contact-info">
+            <p>
+                I'm always <br> interested in 
+                hearing  <br> about new projects, 
+                <br> so if you'd like to <br> chat please 
+                get in  <br> touch.
+            </p>
+        </div>
+        <form action="https://formspree.io/f/mdovbjlz" method="post">
+            <div class="form-group">
+                <input class="input" type="text" name="name" maxlength="30" placeholder="Full name" rows="10" required>
+                <input class="input" type="email" name="email" placeholder="Email address" required>
+                <textarea name="send" maxlength="399" required>Write me something</textarea>
```

Comment:
- [ ] Good job working on "Portfolio: add a contact form and finish the mobile version" 😃, however, your textarea has already the value, so for good practice, it could be better to change this  `<textarea name="send" maxlength="399" required>Write me something</textarea>` to this `<textarea name="send" maxlength="399" required placeholder="Write me something"></textarea>` so that the value you put will be into the `placeholder` 👍 . Kindly, fix it